site/src/sphinx/advanced-structured-logging.rst

```diff
@@ -61,6 +63,92 @@ What properties can be retrieved?
+| ``dnsResolutionDurationNanos``           | the duration took to resolve a domain name, ``-1`` if DNS lookup  |
+|                                          | did not occur                                                     |
++------------------------------------------+-------------------------------------------------------------------+
+| ``socketConnectStartTimeMicros``         | when the client started to connect to a remote peer, in           |
+|                                          | microseconds since the epoch (01-Jan-1970 00:00:00 UTC)           |
++------------------------------------------+-------------------------------------------------------------------+
+| ``socketConnectDurationNanos``           | the duration took to connect to a remote peer, ``-1`` if socket   |
+|                                          | connection attempt did not occur                                  |
++------------------------------------------+-------------------------------------------------------------------+
+| ``pendingAcquisitionStartTimeMicros``    | when the client started to wait for the completion of an existing |
+|                                          | connection attempt, in microseconds since the                     |
+|                                          | epoch (01-Jan-1970 00:00:00 UTC)                                  |
++------------------------------------------+-------------------------------------------------------------------+
+| ``pendingAcquisitionDurationNanos``      | the duration took to wait for the completion of an existing       |
+|                                          | connection attempt to use one connection in HTTP/2, ``-1`` if     |
+|                                          | waiting did not occur                                             |
++------------------------------------------+-------------------------------------------------------------------+
+| ``connectionAcquisitionStartTimeMicros`` | when the client started to acquire a connection, in microseconds  |
+|                                          | since the epoch (01-Jan-1970 00:00:00 UTC)                        |
++------------------------------------------+-------------------------------------------------------------------+
+| ``connectionAcquisitionDurationNanos``   | the duration took to get a connection (i.e. the total duration)   |
++------------------------------------------+-------------------------------------------------------------------+
+
+The total duration is the sum of ``dnsResolutionDurationNanos``, ``socketConnectDurationNanos`` and
+``pendingAcquisitionDurationNanos``. They may or may not occur depending on circumstances.
+These are some of the scenarios how the total duration is composed of:
+
+    1. Resolving a domain name and connecting to the remote peer.
+
+       .. uml::
+
+           @startditaa(--no-separation, --no-shadows)
+
```

Comment:
I'm curious if removing the empty line right after/before `@startditaa/endditaa` will reduce the horizontal margin of the generated images.